src/_guides/language/language-tour.md

```diff
@@ -27,15 +27,15 @@ more details about a language feature.
 
 The following code uses many of Dart’s most basic features:
 
-<!-- language-tour/basic-dart-program/bin/main.dart -->
+<?code-excerpt "language-tour/basic-dart-program/bin/main.dart"?>
 {% prettify dart %}
 // Define a function.
-printNumber(num aNumber) {
+void printNumber(num aNumber) {
```

Comment:
I think, in language tour, we can actually make omissions like this (no `void`) for the sake of smoother beginnings. Does that break the code-excerpt pipeline? If this is because of a lint, maybe we want to use [`// ignore_for_file` annotations](https://www.dartlang.org/guides/language/analysis-options#ignoring-specific-analysis-rules)?

Reply:
> I think, in language tour, we can actually make omissions like this (no void) for the sake of smoother beginnings.

Ok

> Does that break the code-excerpt pipeline?

Not at all.

> If this is because of a lint, ...

Yes. I'm thinking that we have too many lint rules. Maybe we should opt for the same list that we recommend for Angular: https://github.com/dart-lang/stagehand/blob/master/templates/web-angular/analysis_options.yaml

@kwalrath @filiph WDYT?

Reply:
I think in general it's good to have as many lint rules turned on for samples as possible (because by default, samples should be exceptionally "standard" code). I'd rather we keep them on but that we're not afraid to use `// ignore_for_file` where it makes sense.

Reply:
SGTM